### DISCUSSION TITLE
Jeckyl now: HTML Table in the Browser is not correct

### DISCUSSION THREAD

**Discussion by PeterGyger** [community/community#21739] [Other Feature Feedback, Questions, & Ideas]
<p>Hello</p>
<p>On Github and in VSC (Visual Studio Code) the HTML Table locks good</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/d289a0b87d71f5b054eebdf3a27d326abb6efcc6.png" data-download-href="/uploads/short-url/u2v8n30wJrLuCuyw6lEkfi2uFqC.png?dl=1" title="md-tabelle3.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/40018804/181066494-e27ca1f3-da83-4ed2-a919-34beb3a525e1.png" alt="md-tabelle3.png" data-base62-sha1="u2v8n30wJrLuCuyw6lEkfi2uFqC" width="690" height="231" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/d289a0b87d71f5b054eebdf3a27d326abb6efcc6_2_690x231.png, https://aws1.discourse-cdn.com/github/optimized/1X/d289a0b87d71f5b054eebdf3a27d326abb6efcc6_2_1035x346.png 1.5x, https://aws1.discourse-cdn.com/github/original/1X/d289a0b87d71f5b054eebdf3a27d326abb6efcc6.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/d289a0b87d71f5b054eebdf3a27d326abb6efcc6_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">md-tabelle3.png</span><span class="informations">1178×396 25.9 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div> <div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/28bda51dbf4f75f31408ac3c432ee51928dfe165.png" data-download-href="/uploads/short-url/5Oppv8d07rAObudSjhKduzUM9vv.png?dl=1" title="md-tabelle1.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/40018804/181066496-185ac183-9f23-491a-a32b-260362579fa0.png" alt="md-tabelle1.png" data-base62-sha1="5Oppv8d07rAObudSjhKduzUM9vv" width="690" height="221" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/28bda51dbf4f75f31408ac3c432ee51928dfe165_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">md-tabelle1.png</span><span class="informations">1034×332 6.17 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>But not in den  Browsers (MS Edge, Firefox, Valdi - all newest Version). The HTML Tags are not correct displayed. Why? You can see it on the Website:</p>
<p><a href="https://petergyger.github.io/50/" rel="nofollow noopener">https://petergyger.github.io/50/</a></p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/a593fd578ce9b2415f7d0812d26b286ed87ccb7c.png" data-download-href="/uploads/short-url/nCLQDeAw5dkZoik8xVKPoei2GJu.png?dl=1" title="md-tabelle2.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/40018804/181066497-1b7966bf-58c2-4c9c-b552-b75338bd849c.png" alt="md-tabelle2.png" data-base62-sha1="nCLQDeAw5dkZoik8xVKPoei2GJu" width="690" height="219" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/a593fd578ce9b2415f7d0812d26b286ed87ccb7c_2_690x219.png, https://aws1.discourse-cdn.com/github/original/1X/a593fd578ce9b2415f7d0812d26b286ed87ccb7c.png 1.5x, https://aws1.discourse-cdn.com/github/original/1X/a593fd578ce9b2415f7d0812d26b286ed87ccb7c.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/a593fd578ce9b2415f7d0812d26b286ed87ccb7c_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">md-tabelle2.png</span><span class="informations">901×286 14.7 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Ok - i can make a more simple HTML Code for the Table. But then the Table looks not nice.</p>
<p>Who has a idea, what is the Reason for that Behavior?</p>
<p>Thanks for every Idea and Help</p>
<p>Greetings</p>

**Comment by fire-eggs** [comment]
<p>I grabbed a copy of your .md file and tried a few variations in my own Github page area.</p>
<p>I had the best results by deleting the styling you’ve applied to the table columns. E.g. I took this:</p>
<pre><code>&lt;tr style="height: 21px;"&gt;
&lt;td style="padding-left: 5px; padding-right: 5px; border-top: none; border-left: none; border-bottom: none; border-right: solid 0.5pt;"&gt;3&lt;/td&gt;
&lt;td style="padding-left: 5px; padding-right: 5px; border-top: none; border-left: none; border-bottom: none; border-right: solid 0.5pt;" valign="bottom"&gt;&lt;a href="https://de.wikipedia.org/wiki/T%C3%B6ne_des_Hochchinesischen"&gt;mittel nach unten und wieder nach halb hoch&lt;/a&gt;&lt;/td&gt;
&lt;td style="padding-left: 5px; padding-right: 5px; border-top: none; border-left: none; border-bottom: none; border-right: solid 0.5pt;" valign="bottom"&gt;dì sān shēng&lt;/td&gt;
&lt;td style="padding-left: 5px; padding-right: 5px; border-top: none; border-left: none; border-bottom: none; border-right: solid 0.5pt;" valign="bottom"&gt;第三聲&lt;/td&gt;
&lt;td style="padding-left: 5px; padding-right: 5px; border-top: none; border-left: none; border-bottom: none; border-right: solid 0.5pt;" valign="bottom"&gt;第三声&lt;/td&gt;
&lt;td style="padding-left: 5px; padding-right: 5px; border: none;" valign="bottom"&gt;&lt;p style="text-align: center;"&gt;ˇ&lt;/td&gt;&lt;/tr&gt;
</code></pre>
<p>And turned it into this:</p>
<pre><code>&lt;tr&gt;
&lt;td&gt;3&lt;/td&gt;
&lt;td&gt;&lt;a href="https://de.wikipedia.org/wiki/T%C3%B6ne_des_Hochchinesischen"&gt;mittel nach unten und wieder nach halb hoch&lt;/a&gt;&lt;/td&gt;
&lt;td&gt;dì sān shēng&lt;/td&gt;
&lt;td&gt;第三聲&lt;/td&gt;
&lt;td&gt;第三声&lt;/td&gt;
&lt;td&gt;&lt;p&gt;ˇ&lt;/p&gt;&lt;/td&gt;
&lt;/tr&gt;
</code></pre>
<p>And here are the results:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/6/6d4d50dd99a1282b7efc5724f647210d0fcc19c9.png" data-download-href="/uploads/short-url/fAVGX5rgOM6kn8WBn70ao5qp7Hb.png?dl=1" title="Capture.PNG" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/9809727/181066499-3f6d6d6f-1ce4-46c2-97af-560bade3ef9a.png" alt="Capture.PNG" data-base62-sha1="fAVGX5rgOM6kn8WBn70ao5qp7Hb" width="690" height="305" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/6/6d4d50dd99a1282b7efc5724f647210d0fcc19c9_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Capture.PNG</span><span class="informations">983×435 8.5 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>So I don’t know what is going on, but if I compare the “looks good” source in Github, vs the source in Chrome, the Github source doesn’t have all the styling.</p>
<p>I’m not sure what is going on, but this feels like a bug to me - something in the in-line styling is causing the Github Markdown server to fail to generate the &lt;/td&gt;&lt;/tr&gt; tags correctly (they come out as &lt;/tr&gt; etc).</p>
<p>You could try using style sheets for your styling instead of the in-line styling. Or if the third row of the table above “looks OK”, remove the styling altogether. Or try some other variations on the styling, to see what “works”.</p>
<p>Also, I’d submit an issue to the Github team.</p>

**Comment by PeterGyger** [comment, marked as answer]
<p>Problem solved</p>
<p>The 6. Line was a “p” Tag without the closed part. </p>
<p>I delete this Part and now all is fine.</p>
<p>For VSC and GitHub not a Problem. They ignore this fact. </p>
<p>But not for the Browsers.</p>
<p>&lt;td style=“padding-left: 5px; padding-right: 5px; border: none;” valign=“bottom”&gt;&lt;p style=“text-align: center;”&gt;</p>

**Comment by fire-eggs** [comment]
<p>Ha! It’s always something simple! Good eye!</p>

**Comment by PeterGyger** [comment]
<p>You are very kind.</p>
<p>I’don’t understand my mistake.</p>
<p>And ich have to learn more about VSC with HTML / Markdown</p>
<p>I’m sure, i can check the HTML Code with VSC and prevent Errors like this</p>
